src/Svelte/assets/dist/render_controller.js

```diff
@@ -37,6 +37,13 @@ class default_1 extends Controller {
         };
         this.dispatch(name, { detail, prefix: 'svelte' });
     }
+    async mountSvelteComponent(Component, options) {
+        const { mount } = await import('svelte');
```

Comment:
This will not work with AssetMapper right ? In which cas we need to find a solution because this would be a giant BC break :)

Reply:
That was my first thought but it worked when I tried it in ux.symfony.com !
I also tried to have `import { mount } from 'svelte'` at the top of the file but it doesn't work with AssetMapper with Svelte 4, an error was thrown because `mount `only exists in v5
I can't explain why but with a dynamic import there is no error, mount is just `undefined`

Reply:
I'll test this next week, thanks for the explanations :)

Reply:
Nice! FYI with AssetMapper it works with Svelte 4, but with Svelte 5 I'm stuck with the issue described [here](https://github.com/sveltejs/svelte/issues/13186#issuecomment-2341441252) 
I made a [REPL](https://codesandbox.io/p/sandbox/xtvzpy?file=%2Findex.html%3A20%2C1) with only html/js that reproduce the error